### DISCUSSION TITLE
circuit_breaking_exception on standalone install

### DISCUSSION THREAD

**Discussion by chadlangston** [Security-Onion-Solutions/securityonion#2031] [Unsupported Versions] [answered]
Does anyone know how to solve this?

circuit_breaking_exception: [parent] Data too large, data for [indices:data/read/search] would be [607325686/579.1mb], which is larger than the limit of [597688320/570mb], real usage: [607324208/579.1

This is a vanilla standalone install. Google search suggests that it is related to elasticsearch. so-status says everything is ok. I get this error in Alerts & Hunt, and Kibana displays a 404 not found.

Thanks for any help or insights!

Best, 
-Chad

**Comment by weslambert** [comment, marked as answer]
Looks like you need to adjust the heap for ES in `/opt/so/saltstack/local/pillar/minions/$minion.sls`, then restart ES with `so-elasticsearch-restart`

**Comment by chadlangston** [comment]
Brilliant!

Thank you Wes! I will spend some more time reviewing the settings in that
file and trying to understand the whole Salt architecture better.

While I am learning it would be helpful if there was an easy way to reset
the whole standalone system without having to rebuild the machine. I am
running the network install. The last time I tried to re-install
SecurityOnion using so-setup-network it failed. I had tried doing some
brute force things, like deleting everything in /nsm first, but I ended up
getting Salt errors. So, if there are some instructions on how to delete
everything so I can re-run the so-setup-network script again without it
failing, that would be awesome!

It's a headless machine, so it is a pain for me to rebuild the whole
machine with a clean OS.

Thanks Wes!

Chad
